+nosnoc/+dcs/Heaviside.m

```diff
@@ -129,7 +132,7 @@ function generate_equations(obj, opts)
 
             obj.f_x_fun = Function('f_x', {model.x, model.z, obj.alpha, obj.lambda_n, obj.lambda_p, model.u, model.v_global, model.p}, {obj.f_x, model.f_q});
             obj.f_q_fun = Function('f_q', {model.x, model.z, obj.alpha, obj.lambda_n, obj.lambda_p, model.u, model.v_global, model.p}, {model.f_q});
-            obj.g_z_fun = Function('g_z', {model.x, model.z, model.u, model.v_global, model.p}, {model.g_z});
+            obj.g_z_fun = Function('g_z', {model.x, obj.alpha, model.z, model.u, model.v_global, model.p}, {model.g_z});
```

Comment:
here it is a bit hard to follow without comments. 
Once again, g_z are the user algebaric equations, and g_alg_ are the DCS algebarics (e.g. lp equations) - please clarify and comment a bit.
And why are all variables from the model, only the alpha from the object (comment also briefly in code)

Reply:
Technically these are all documented (though probably need to be _better_ documented) in `nosnoc.dcs.Base`. I can add some brief documentation though I am always wary of documenting in multiple places as this has a tendency to run into desyncing issues.